### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by allencloud**
fix typos

Signed-off-by: allencloud <allen.sun@daocloud.io>

### CODE REVIEW

File: vendor/gopkg.in/yaml.v2/scannerc.go [hunk 1]
```diff
@@ -1618,7 +1618,7 @@ func yaml_parser_scan_directive_name(parser *yaml_parser_t, start_mark yaml_mark
 		return false
 	}
 
-	// Check for an blank character after the name.
+	// Check for a blank character after the name.
```
Comment (dmcgowan) [inline vendor/gopkg.in/yaml.v2/scannerc.go]: This change needs to be removed, vendor changes must be done upstream

**Comment by allencloud** [comment]
Thanks. Updated @dmcgowan

**Comment by dmcgowan** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM (not a maintainer here) docs failure is not related

**Comment by RichardScothern** [comment]
LGTM

**Pull request merged**